pkg/util/util_test.go

```diff
@@ -332,3 +334,93 @@ func TestJoinQualifiedName(t *testing.T) {
 		}
 	}
 }
+
+const fileData = `Iface	Destination	Gateway 	Flags	RefCnt	Use	Metric	Mask		MTU	Window	IRTT                                                       
```

Comment:
I think we need to define a set of interesting test cases and comment why they are interesting.

"normal" with default route first
"normal" with default route last
no default route
etc...

I want to be confident this test covers as much of the error handling as humanly possible.